### ISSUE TITLE
blue screens?

### ISSUE THREAD

**Issue by muellerto** [valinet/ExplorerPatcher#3541]
I have the impression Explorer Patcher was responsible for a lot of blue screens on my machine.

This came suddenly after Windows updates five days ago and didn't stop until I decided today to uninstall EP. Now, without EP, my machine seems to be rather stable again. Didn't have a single blue screen since then.

The EP version which has originally been installed was the last one which came by update. (I mean when you click on "search for updates" inside EP. Sorry, I have no better number.) Because this one was probably already some weeks old I replaced it by the latest one from the releases section here, 22621.3880.66.4, but the problem remained.

I don't know how early EP is involved in the boot process but sometimes I got a blue screen already before login, sometimes during login (entering my password) and sometimes later. It happened that I could work for two or three hours but then it came again. I could never make a concrete user action directly responsible for that, sometimes I wasn't even there getting a coffee in the kitchen and when I came back I had a blue screen. Two times it happened that I got into a boot loop, you get then three or four blue screens one after the other.

The blue screens themselves do never say much. The so called stop codes were mostly `SYSTEM_SERVICE_EXCEPTION`, also `CRITICAL_PROCESS_DIED` and once `PAGE_FAULT_IN_NONE_PAGED_AREA`.

My OS is Windows 11 Pro, version 23H2 (Build 22631.3958), German speaking.

**Comment by pyrates999** [comment]
I would try to repair your windows install.  Please run the following in an elevated command prompt or elevated powershell:

```
chkdsk C:
```

If it tells you there are errors, then run this:

```
chkdsk C: /F
```

Reboot to run it on boot.  Keep running that command until it reports no errors.  Then run this to repair windows:

```
sfc /scannow
dism /online /cleanup-image /scanhealth
dism /online /cleanup-image /restorehealth
dism /online /cleanup-image /checkhealth
dism /online /cleanup-image /analyzecomponentstore
dism /online /cleanup-image /startcomponentcleanup
```

Then reboot your computer.

**Comment by Amrsatrio** [comment]
I cannot say exactly regarding blue screens especially those three you mentioned, in my case I keep getting DPC_WATCHDOG_VIOLATION which happens only when my CPU is under load. I use an i9-12900K by the way. Other than that I haven't been getting them on my other devices: MacBook Pro 2020 (i7-1068NG7), Lenovo IdeaPad Duet 5i (i7-1255U), Lenovo Yoga Slim 7x (Snapdragon X Elite 78)

**Comment by Amrsatrio** [comment]
Are you still experiencing bluescreens? Maybe try a clean install or re-seating your components especially RAM in case of a desktop?

**Comment by muellerto** [comment]
I made a complete Windows reset in the last days :( and had to reinstall all my software. The endless loop of bluescreens did damage my disk and my machine didn't boot anymore at all.

This is the reason why the situation is nor different. Also my system is different now, it's a clean Windows 11 now, the previous one has been upgraded from Windows 10. I did also reinstall the latest release of EP and it runs good. I never had a new bluescreen. So I guess EP was not the only reason why my machine crashed again and again, later (after my post) it did also crash without EP running, so there must have been something more. But we'll never find out what this has been. That's why I close the case here.
